### PR TITLE
Added binary search

### PR DESCRIPTION
**Pull request by neeru-1997**
Hi 

I have added Binary Search in Go Language under src folder.

Regards

### CODE REVIEW

File: src/binarySearch.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Binary Search in Golang
+package main
+import "fmt"
+
```
Comment (rajkumarGosavi) [inline src/binarySearch.go:4]: Please provide a comment for the functionality of the code.

**Review by rajkumarGosavi — COMMENTED** [review]
The snippet should be commented properly, so that the learner will be able to understand the flow

**Comment by sagar-jadhav** [comment]
@neeru-1997 Can you please have a look on review comments? & Incorporate necessary changes.

**Comment by sagar-jadhav** [comment]
@neeru-1997 any updates on **incorporating review comments**? We have to close this as early as possible.

Let me know if you required any help from our side.

**Comment by sagar-jadhav** [comment]
@neeru-1997 I am closing the #37 PR due to inactivity. Please raise another PR once you are ready with the changes.

**Pull request closed without merging**